src/sentry/integrations/metric_alerts.py

```diff
@@ -74,7 +75,9 @@ def incident_attachment_info(incident, metric_value=None, action=None, method=No
     else:
         metric_and_agg_text = f"{metric_value} {agg_text}"
 
-    text = f"{metric_and_agg_text} in the last {time_window} minutes"
+    interval = "minute" if time_window == 1 else "minutes"
+
+    text = _(f"{metric_and_agg_text} in the last {time_window} {interval}")
```

Comment:
I don't think you can use ugettext like this, it needs to be templatized.

Reply:
:( okay let me try again, idk why but the test I wanna use this on is just freaking out locally so I was hoping CI would catch stuff.